plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -210,8 +214,76 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 					.errorStatus("Expected location of minishift command does not exist: " + minishiftLoc
 							+ "\nPlease set a correct value in the server editor."));
 		}
+		return minishiftLoc;
+	}
+	
+	private void launchCheckSetupCDK(IServer s, CDKServer cdkServer, ControllableServerBehavior beh) throws CoreException {
+		CDK3Server cdk3 = (CDK3Server)cdkServer;
+		if( !cdk3.isCDKInitialized()) {
+			int[] retmain = new int[1];
+			retmain[0] = -1;
+			String home = cdk3.getMinishiftHome();
+			Display.getDefault().syncExec(() -> {
+				MessageBox messageBox = new MessageBox(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell(), SWT.ICON_WARNING | SWT.OK | SWT.CANCEL);
```

Comment:
Please, change MessageBox to MessageDialog: new MessageDialog(shell, title, null, msg, MessageDialog.WARNING, new String[] {"OK", "Cancel"}, 0);